### ISSUE TITLE
Investigate which software embeds zcashd

### ISSUE THREAD

**Issue by daira** [zcash/zcash#2305]
This is needed because users of such software (e.g. miners and external wallets) may not be as aware that they need to upgrade, as users who have installed zcashd explicitly.

**Comment by radix42** [comment]
yes, this is why I expressed a desire for some kind of clear error code or something in json returned to RPC calls or by zcash-cli if the node has gotten a "you're too old, upgrade now, you're running in a degraded mode" message, so I can pop that info up to the user in zcash4mac and zcash4win
